app/routes/companyInterest/components/CompanyInterestPage.tsx

```diff
@@ -449,12 +318,22 @@ type Props = {
   initialValues: any;
 };
 
+const requiredIfEventType = (eventType: string) =>
+  requiredIf(
+    (allValues) =>
+      allValues.events.filter((event) => event.name === eventType)[0].checked
+  );
+
 const validate = createValidator({
   company: [required()],
   contactPerson: [required()],
   mail: [required(), isEmail()],
   phone: [required()],
-  comment: [required()],
+  aboutTheCompany: [required()],
+  companyType: [required()],
+  officeInTrondheim: [required()],
+  events: [required()],
+  breakfastTalkComment: [requiredIfEventType('breakfast_talk')],
 });
```

Comment:
Seems like you might have forgotten to validate the comments of the other event types